streampipes-extensions/streampipes-processors-geo-jvm/src/main/java/org/apache/streampipes/processors/geo/jvm/jts/processor/latlngtojtspoint/LatLngToJtsPointProcessor.java

```diff
@@ -101,7 +101,11 @@ public void onEvent(Event event, SpOutputCollector collector) throws SpRuntimeEx
     Point geom = SpGeometryBuilder.createSPGeom(lng, lat, epsg);
 
     if (!geom.isEmpty()) {
-      event.addField(WKT_RUNTIME, geom.toString());
+      // if activated the stream fails at all
+      //event.removeFieldBySelector(latitudeMapper);
+      //event.removeFieldBySelector(longitudeMapper);
```

Comment:
Can we please refrain from committing dead code? :)

Reply:
it's not dead, it is in discussion :)
https://github.com/apache/streampipes/discussions/1276
it is not working currently but should be part of the processor

Reply:
Well then this PR isn't ready to be merged right?

Reply:
It is ready to merge. The pull request is for renaming the runtime names.
This code snipped is just a reference for the discussion.
Everything else is still working like before.
It is only a nice to have implementation but not required and can be resolved in a new PR after the discussion is finished :)

Reply:
Frankly speaking, I don't think it's a good idea to commit commented code for several reasons.
Alternatively, if you want to have a reference for your discussion you could create a gist, simply paste the code snippet into the discussion, or create a dedicate branch for it.
But I don't want to get too hung up on two lines of code here right now ✌🏼 
So if you want to do it that way, feel free 🙂

Reply:
Yes, it's good that you care so much about clean code. I have removed it.